projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget-column.model.ts

```diff
@@ -51,6 +58,12 @@ export class TableWidgetColumnModel {
   })
   public alignment?: TableCellAlignmentType;
 
+  @ModelProperty({
+    key: 'classes',
+    type: ARRAY_PROPERTY.type
+  })
+  public classes: string[] = [];
```

Comment:
How would I know what classes I can add? Every cell renderer implementation may have its own set.

Reply:
They would need to be global or available in the renderer.